rank/services/search-templates.ts

```diff
@@ -21,10 +21,12 @@ export async function listIndices(): Promise<Index[]> {
   return indices
 }
 
-export async function getProductionQueries() {
-  const res = await fetch(
-    'https://api.wellcomecollection.org/catalogue/v2/search-templates.json'
-  )
+async function getLiveQueries(environment) {
```

Comment:
Could we type this argument to `QueryEnv`? This would have caught the bug below.

Reply:
`getEnvironmentQueries`?

Reply:
"caught the bug below"

Which bug?